CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 
 ### Jsonnet
 
+* [ENHANCEMENT] Memberlist now uses DNS service-discovery by default. #2549
```

Comment:
Order should be CHANGE, FEATURE, ENHANCEMENT.

Reply:
fixed in https://github.com/grafana/mimir/pull/2549/commits/efd29664b97c7aaa6f818bf3b9afa4981f4b17a8